forge/db/migrations/20250401-01-EE-add-topicPrefix-to-brokercredentials.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-unused-vars */
+
+/*
+ * Adds the topicPrefix column to the BrokerCredentials Table
+ */
+const { DataTypes, QueryInterface } = require('sequelize')
+
+module.exports = {
+    up: async (context) => {
+        await context.addColumn('BrokerCredentials', 'topicPrefix', {
+            type: DataTypes.STRING,
+            defaultValue: '["#"]',
+            allowNull: true,
+            get () {
+                const rawValue = this.getDataValue('topicPrefix')
+                if (rawValue) {
+                    return JSON.parse(rawValue)
+                } else {
+                    return ['#']
+                }
+            },
+            set (value) {
+                if (value) {
+                    this.setDataValue(this.topicPrefix, JSON.stringify(value))
+                }
+            }
+        })
+    },
```

Comment:
```suggestion
    /**
     * Add topicPrefix field to BrokerCredentials table
     * @param {QueryInterface} context Sequelize.QueryInterface
     */
    up: async (context) => {
        await context.addColumn('BrokerCredentials', 'topicPrefix', {
            type: DataTypes.TEXT,
            defaultValue: '["#"]',
            allowNull: true
        })
    },
```

get and set are "Model" properties / application level things i.e.. part of the model's logic, not the database structure.

Example:
* DeviceSettings.value migration: https://github.com/FlowFuse/flowfuse/blob/26925ab21990bc6360f3d9360771e289e45f2ce0/forge/db/migrations/20220624-01-add-devicesettings.js#L23-L24
* DeviceSettings.value model: https://github.com/FlowFuse/flowfuse/blob/26925ab21990bc6360f3d9360771e289e45f2ce0/forge/db/models/DeviceSettings.js#L17-L18


Also, jsdoc type hints are very useful here. 

Lastly, I changed the type to `TEXT`.  Many of our other JSON fields are type TEXT and I read somewhere that some databases default to 255 characters for the STRING type?